### PR TITLE
Stupid Sexy Survivor: Synthetic fabrics

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
```SUMMARY: Content "Introduces synthetic fabrics as a material"```

#### Purpose of change
Previously, one could cut up a bunch of plastic clothing and make impermeable object out of them, such as canteens and jugs.

#### Describe the solution
2 new fabrics have been made: `lycra`, which is mostly nylon/polyethylene terephthalate(PET) mixed with polyurethane based elastomers, and `synthetic fabric`, a split between nylon/polyethylene terephthalate(PET).  Anywhere one would want stretchy, form fitting fabric, use `lycra`. Anywhere else where strong nylon or versatile PET is fine, use `synthetic fabric`.  Where impermeable plastics are used, like those in rain ponchos, rain coats, rubber boots, clean suits and hazmat suits, the original plastic will be used.

2 new materials requirement sets have been added: `"fabric_standard_permeable"`, for anything that you'd think of as 'normal clothing' like shirts, pants and the like, and `"fabric_standard_nostretch"` for any looser fitting clothing.

I have also added a number of sources for lycra, which are fun in and of themselves: unitards, leotards, and zentai (whole body stocking).  
I've added recipes for many of these have been added as well, mostly based on cotton counterparts. The overlooked belly holster has gotten a recipe too.

As replacing many of the 'rag' components of the recipes would be a GIANT pr, I've left that out for now.  No items or recipes touch 'synthetic fabric' just yet.
#### Describe alternatives you've considered
Not having a separate stretchy fabric. (stretches making compression garments from woven nylon though)

#### Testing
Loaded the game a couple times and spawned some of the items, compared the new items against the old.
#### Additional context

More chemically specific kinds of polymers assists in future recipes and such.

'Plastic' can now  largely be considered: LD/HD/UHMW-Polyethylene, ABS plastic, epoxy resins, some PLA parts, phenolic resins (bakelite, lab bench tops, industry-type bottle caps) and polypropylene. These are all more 'structural' plastics, good for making solid things.

'Snythetic fabric' can now broadly be considered to be woven nylon-type polymers, woven polyester(PET), and to a lesser extent, rayon.

This PR began life as a simple recipe to make deep concealment holsters.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -9610,5 +9645,39 @@
...
+    "type": "ARMOR",
+    "name": "zentai",
+    "description": "Short for /zenshin taitsu/, the zentai is a form fitting full body nylon and spandex garment.  Though it covers your face, you can see through the integrated hood.",
+    "weight": "300 g",
+    "volume": "500 ml",
+    "price": 2500,
+    "material": [ "lycra" ],
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "HEAD", "MOUTH", "ET", "TORSO", "ARMS", "HANDS", "LEGS", "FEET" ],
```
Comment (Rail-Runner) [inline data/json/items/armor.json]: What is "ET" here?

File: data/json/items/resources/plastic.json [hunk 2]
```diff
@@ -13,6 +13,35 @@
     "color": "light_blue",
     "flags": [ "NO_SALVAGE" ]
   },
+  {
+    "id": "nylon",
+    "type": "TOOL",
+    "category": "spare_parts",
+    "name": "plastic chunk",
```
Comment (Rail-Runner) [inline data/json/items/resources/plastic.json]:
```suggestion
    "name": "nylon patch",
```

File: data/json/materials.json [hunk 3]
```diff
@@ -1103,6 +1131,34 @@
     "bash_dmg_verb": "dented",
     "cut_dmg_verb": "scratched"
   },
+  {
+    "type": "material",
+    "ident": "nylon",
+    "name": "synthetic fabric",
```
Comment (Rail-Runner) [inline data/json/materials.json]:
```suggestion
    "name": "Synthetic Fabric",
```

File: data/json/recipes/armor/torso.json [hunk 4]
```diff
@@ -979,6 +1003,19 @@
...
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_TORSO",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "time": "42 m",
+    "autolearn": true,
+    "using": [ [ "sewing_standard", 8 ] ],
+    "components": [ [ [ "lycra_patch", 7 ] ] ]
+  },
+  {
```
Comment (Rail-Runner) [inline data/json/recipes/armor/torso.json]:
```suggestion
```

**Comment by Rail-Runner** [comment]
Might also want to add new materials to list of materials that tailoring tools can repair, so that there is a way to repair/refit those items.

File: data/json/items/armor.json (hunk 1 above)
Comment (Night-Pryanik) [reply data/json/items/armor.json]: Zentai covers aliens, nothing unusual.

Comment (tenmillimaster) [reply data/json/items/armor.json]: oh, dammit. I thought I fixed that. I always mess up during fixing merge conflicts.

**Comment by tenmillimaster** [comment]
Thought I had; sorry, my other machine seems to not have pushed to my repo correctly or something.

**Comment by tenmillimaster** [comment]
Closing; I need to re-review everything since my other machine's local isn't matching what I pushed this morning. And also find out why it's not seeing commits on remote.

**Pull request closed without merging**